### PR TITLE
Fixes space heaters so that they can space heat again (and space cool)

### LINKED ISSUE #84591: Space heaters have a negligible effect
Reporting client version: 515.1636

## Issue Summary

In the past, temperature differences resulting from space heaters were much stronger, but coenciding with the bug that caused [space heaters to drain power immensely quick](https://github.com/tgstation/tgstation/issues/84440), this issue was introduced as well.
As it currently stands, space heaters have little to no effect, even in extreme temperature differences.

## Round ID:
[231512](https://scrubby.melonmesa.com/round/231512)

## Reproduction:

1. Place down a space heater in a hallway and upgrade it to tier four parts (Capacitor + laser)
2. Unscrew the cover of the space heater, and set it to max / min temperature possible.
3. Observe as nothing happens (atmos thermal googles recommended)

### PR DESCRIPTION
**Pull request by Metekillot**
## About The Pull Request

Space heater efficiency didn't get adjusted with the megacellening and that's what this PR does.
## Why It's Good For The Game

Without the space heater how will I space cool and possible space heat a room?
Fixes #84591 
## Changelog
:cl: Bisar
fix: Space heaters are now able to space cool and heat at their previous space heating and cooling rate.
/:cl:

### CODE REVIEW

**Comment by SyncIt21** [comment]
Adjust the examine as well.

https://github.com/tgstation/tgstation/blob/360d69752141d80e7c7daaa92405bedee270ea22/code/game/machinery/spaceheater.dm#L108

Also can we use a define instead of 1e4? Maybe write in terms of `STANDARD_CELL_RATE`.

File: code/game/machinery/spaceheater.dm [hunk 1]
```diff
@@ -2,7 +2,8 @@
 #define HEATER_MODE_HEAT "heat"
 #define HEATER_MODE_COOL "cool"
 #define HEATER_MODE_AUTO "auto"
-#define BASE_HEATING_ENERGY (STANDARD_CELL_RATE * 0.1)
```
Comment (SyncIt21) [inline code/game/machinery/spaceheater.dm:5]: Don't change this. STANDARD_CELL_RATE is a global value that is shared by a lot of cell devices.

 Using your own constant value will break the space heater If someone changes it's value in the future.

I didn't mean create your own define, but use this pre existing one

**Review by SyncIt21 — CHANGES_REQUESTED** [review]

File: code/game/machinery/spaceheater.dm [hunk 2]
```diff
@@ -2,7 +2,8 @@
...
+#define BASE_HEATING_ENERGY (40 KILO JOULES)
+#define SPACE_HEATING_COEFFICIENT 1e4
```
Comment (SyncIt21) [inline code/game/machinery/spaceheater.dm:6]: Write this value 1e4 in terms of STANDARD_CELL_CHARGE

**Comment by Metekillot** [comment]
I'm not feeling the juju to finish this up so I'll leave it in someone else's hands.

**Pull request closed without merging**